packages/perseus/src/widgets/interactive-graphs/graphs/angle.test.tsx

```diff
@@ -1,9 +1,8 @@
 import {render, screen} from "@testing-library/react";
 import * as React from "react";
 
-import {Dependencies} from "@khanacademy/perseus";
-
 import {testDependencies} from "../../../../../../testing/test-dependencies";
+import * as Dependencies from "../../../dependencies";
```

Comment:
This is a `*` import because that's how we export it in `packages/perseus/src/index.ts`.